devops/ansible/examples/girder-dev-environment/site.yml

```diff
@@ -4,7 +4,7 @@
   vars:
     girder_update: no
     girder_force: no
-    girder_virtualenv: "{{ ansible_user_dir }}/.virtualenvs/girder"
+    girder_virtualenv: "{{ ansible_user_dir }}/virtualenvs/girder"
```

Comment:
I believe the `.virtualenvs` directory is a convention of [virtualenvwrapper](https://bitbucket.org/dhellmann/virtualenvwrapper), however I don't think the wrapper is installed by our provisioning scripts.

Reply:
I suppose the sourcing of the virtual environment is the essential part of this PR. I just found it confusing when I would `vagrant ssh` in. I didn't know girder was installed into a virtualenv, nor is the virtualenv visible right away. 

Automatically sourcing the virtualenv makes it simpler. Would you ever want to ssh into it and not be in the virtualenv?

Reply:
I'm all for auto-sourcing the virtualenv. I'd just probably leave the directory as `.virtualenvs` since that's a common convention.